### PR TITLE
Add endpoints for bugsnag-cli testing

### PR DESCRIPTION
**Pull request by joshedney**
## Goal

Add additional sourcemaps endpoints so that we can use MazeRunner to test the Bugsnag CLI.

## Changeset

Add the following endpoints to the mock server:
`/dart-symbol`
`/ndk-symbol`
`/proguard`

## Tests

Updated tests, covered by CI

### CODE REVIEW

**Comment by twometresteve** [comment]
No objection as such, but could the CLI be configured to send to a different endpoint instead?

Also, please add a changelog entry.

**Comment by joshedney** [comment]
> No objection as such, but could the CLI be configured to send to a different endpoint instead?
> 
> Also, please add a changelog entry.

We can overwrite the server address but the endpoints themselves are hardcoded as these don't change regardless if its SASS or on prem. We could look at adding a `endpoint` flag but this would only be used for interacting with MR. Which would you prefer ?

File: lib/maze.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # Glues the various parts of MazeRunner together that need to be accessed globally,
 # providing an alternative to the proliferation of global variables or singletons.
 module Maze
-  VERSION = '8.1.4'
+  VERSION = '8.1.5'
```
Comment (twometresteve) [inline lib/maze.rb:10]: I think I'd got for 8.2.0 - whilst minor, it does constitute and Enhancement rather than a Fix.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 8.1.5 - TBD
+
+- Add additional sourcemap endpoints [570](https://github.com/bugsnag/maze-runner/pull/570)
```
Comment (twometresteve) [inline CHANGELOG.md:3]: Please add an `## Enhancements` header.

**Review by twometresteve — APPROVED** [review]

**Pull request merged**